bugtool/configuration/cilium.go

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package configuration
+
+import (
+	dump "github.com/cilium/cilium/bugtool/dump"
+)
+
+func CiliumTasks() dump.Tasks {
+	ts := dump.Tasks{}
+	for _, cmd := range ciliumCommandsJSON() {
+		ts = append(ts, createExecFromString(cmd, "json"))
+	}
+	return ts
+}
+
+func ciliumCommandsJSON() []string {
+	var commands []string
+	generators := []func() []string{
+		CiliumAgentAPICLICommands,
+	}
+
+	for _, generator := range generators {
+		commands = append(commands, generator()...)
+	}
+
+	return commands
+}
+
+// CiliumAgentAPICLICommands returns a list of all dump commands that use the
+// Ciliums Agent CLI binary to dump data from the Agents API.
+func CiliumAgentAPICLICommands() []string {
+	commands := []string{
+		"cilium debuginfo --output=json", // debuginfo uses different output flag format.
+	}
+	// TODO: Many of these are redundant with debuginfo, clean them up.
+	for _, cmd := range []string{
+		"cilium metrics list",
+		"cilium fqdn cache list",
+		"cilium config -a",
+		"cilium encrypt status",
```

Comment:
This one doesn't seem to support a proper JSON output; it outputs text.

Reply:
Moved it to ".md" file for now.  I'll add it to the list of commands to fix.